utiltydb.cpp

```diff
@@ -0,0 +1,91 @@
+#include "utiltydb.h"
+
+UtilityDB::UtilityDB() { establishConnection(); }
+
+bool UtilityDB::isConnected() { return db.open(); }
```

Comment:
This method is defined but never used.

Reply:
Because I didn't specify any use of UtilityDb API.  That method could be used before making requests to the DB. I'm  also going to remove `QMessageBox` messages in method `establishConnection`.
So the developers will use this method to check the connection and only then performs some needed actions.

Reply:
OK, then maybe [this method](https://doc.qt.io/qt-5/qsqldatabase.html#isOpen) should be used instead?

Reply:
Make sense 👍